src/main/webapp/createYourOwn.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>altruism-ator</title>
+    <link rel="stylesheet" href="create-your-own-style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Karla&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <header>
+        <div class="container">
+            <nav>
+                <ul>
+                    <li class="nav-item"><a href="index.html">Home</a></li>
+                    <li class="nav-item"><a href="profile.html">Profile</a></li>
+                    <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
+                    <li class="nav-item"><a href="chat.html">Chat</a></li>
+                </ul>
+            </nav>
+        </div>
+      </header>
+  </body>
+</html>
```

Comment:
Add trailing newline please.

Reply:
If you could update the title of the page.

Reply:
Same comment as above, I added the trailing line. However, for the title, I think it is better to keep the name of our page overall rather than just "Create Your Own"